lib/adapters/webdriverio.js

```diff
@@ -26,10 +26,13 @@ WebDriverIOAdaptor.prototype = objectAssign({}, Browser, {
 
   getBoundingClientRect: function(selector, callback) {
     var webdriverio = this._webdriverio;
-
```

Comment:
Can you add this newline back? It's easier to read if the var statement and the code below are separated.